contentcuration/contentcuration/tests/test_channel_views.py

```diff
@@ -51,6 +52,7 @@ def test_no_public_channels(self):
         """
         Ensure that if there are no public channels, we get 0 results from the serializer.
         """
+        cache.clear()
```

Comment:
One change I had to make is that, since our cache no longer relies on a db backend, it wasn't getting wiped with each test. I looked into ways of clearing the cache only for this specific page, but using variations on the API endpoint name as the key did not work. Will gladly update this to only delete the page cache if someone knows how to do this! :)

Reply:
Good catch! Should we do this in a `tearDown` though, to ensure each test gets a cleared cache?

Reply:
Sounds good, I made the change!